public/assets/js/game.js

```diff
@@ -282,6 +282,7 @@ $(function () {
     // Axios to get the game data
     getCards($gameTeamSelection.val(), $gameQuestionSelection.val());
     // Show game instructions
+    $gameForm.css("display", "none");
```

Comment:
I intentionally had those dropdowns stay there so they could see the question and team drop-downs while playing the game. So I'm wondering why we're now going to hide them? Also if they wanted to change the question when the instructions come up, hiding that section would not allow them to do that.

Reply:
Change removed